src/api/libcellml/annotator.h

```diff
@@ -840,25 +829,22 @@ class LIBCELLML_EXPORT Annotator: public Logger
     std::string assignId(const VariablePairPtr &pair, CellmlElementType type = CellmlElementType::MAP_VARIABLES);
 
     /**
-     * @overload
-     *
      * @brief Assign an automatically generated, unique identifier to the item of type @p type
      *        which exists between @p item1 and @p item2.
      *
+     * @overload
+     *
      * This method will return the new identifier that has been assigned, or an empty string
      * if the operation failed.
      * The default is to assign an identifier to a MAP_VARIABLES item.
      *
      * Only CellmlElementType::MAP_VARIABLES, and CellmlElementType::CONNECTION are relevant any other
      * CellmlElementType type will not assign an identifier.
      *
-     * @see assignId(const AnyCellmlElementPtr &item) for a list of reasons that an identifier may not be assigned.
-     * @see errorCount() if and/or error(size_t) for any issues that may have been raised.
+     * @see assignId(const AnyCellmlElementPtr &item)
      *
-     * @overload
-     *
-     * @param variable1 A @ref VariablePtr item defining to the first item in a connection or equivalence.
-     * @param variable2 A @ref VariablePtr item defining to the second item in a connection or equivalence.
+     * @param variable1 A @ref VariablePtr item defining to the first item in a equivalence.
+     * @param variable2 A @ref VariablePtr item defining to the second item in a equivalence.
```

Comment:
"... defining the..." rather than "defining TO the..."

Reply:
a7ccc63